scripts/installscripts/install-jukebox.sh

```diff
@@ -1082,40 +1089,25 @@ install_main() {
     sudo chown mpd:audio "${mpd_conf}"
     sudo chmod 640 "${mpd_conf}"
 
-    # start mpd
```

Comment:
Where do you start MPD now?

Reply:
MPD and other services will start after the reboot, since there have been `enabled`.